pandas/core/indexes/multi.py

```diff
@@ -2862,10 +2862,11 @@ def get_loc_level(self, key, level: IndexLabel = 0, drop_level: bool = True):
 
         Returns
         -------
-        loc : A 2-tuple where the elements are:
-              Element 0: int, slice object or boolean array
-              Element 1: The resulting sliced multiindex/index. If the key
-              contains all levels, this will be ``None``.
+        tuple
```

Comment:
Can you render this and check how it looks?

Reply:
> Can you render this and check how it looks?

Yes, that is not pretty, and I make some changes, it looks like this now.

![image](https://user-images.githubusercontent.com/77875500/207219196-b7607cea-092c-47d2-8457-dd1eb436caf8.png)